src/components/MediaSection/index.tsx

```diff
@@ -26,193 +31,182 @@ export default function MediaSection({ childrenPosts }: IMediaSection) {
-            return !prev.includes(item.data) ? [...prev, item.data] : [...prev]
-          }
-          );
+            return !prev.includes(item.data) ? [...prev, item.data] : [...prev];
+          });
         }
       });
     } catch (error) {
       console.log('error: ', error);
     }
-  }
-
-
+  };
 
   useEffect(() => {
-
     getPostInfo();
-
-  }, [childrenPosts, currentPostdetail, postList, postListGlobal])
+  }, [childrenPosts, currentPostdetail, postList, postListGlobal]);
 
   function onClickImage(index: number): void {
     setIsVisibleFullImage(true);
     setImageIndex(index);
   }
 
-
-
-
-
-  function renderMediaPosts()  {
+  function renderMediaPosts() {
     const thumbnailFileIds: string[] =
       videoPosts.length > 0
         ? videoPosts.map((item) => {
-          return `https://api.${apiRegion}.amity.co/api/v3/files/${item?.thumbnailFileId}/download?size=medium`;
-        })
+            return `https://api.${apiRegion}.amity.co/api/v3/files/${item?.thumbnailFileId}/download?size=medium`;
+          })
         : [];
-    let mediaPosts: string[] = []
-    mediaPosts = [...imagePosts].length > 0 ? [...imagePosts] : [...thumbnailFileIds];
-    const imageElement = mediaPosts.map(
-      (item: string, index: number) => {
-        if (mediaPosts.length === 1) {
-          imageStyle = styles.imageLargePost;
-          colStyle = styles.col6;
-        } else if (mediaPosts.length === 2) {
-          colStyle = styles.col3;
-          if (index === 0) {
-            imageStyle = [styles.imageLargePost, styles.imageMarginRight];
-          } else {
-            imageStyle = [styles.imageLargePost, styles.imageMarginLeft];
-          }
-        } else if (mediaPosts.length === 3) {
-          switch (index) {
-            case 0:
-              colStyle = styles.col6;
-              imageStyle = [styles.imageMediumPost, styles.imageMarginBottom];
-              break;
-            case 1:
-              colStyle = styles.col3;
-              imageStyle = [
-                styles.imageMediumPost,
-                styles.imageMarginTop,
-                styles.imageMarginRight,
-              ];
-              break;
-            case 2:
-              colStyle = styles.col3;
-              imageStyle = [
-                styles.imageMediumPost,
-                styles.imageMarginTop,
-                styles.imageMarginLeft,
-              ];
-              break;
-
-            default:
-              break;
-          }
+    let mediaPosts: string[] = [];
+    mediaPosts =
+      [...imagePosts].length > 0 ? [...imagePosts] : [...thumbnailFileIds];
+    const imageElement = mediaPosts.map((item: string, index: number) => {
+      if (mediaPosts.length === 1) {
+        imageStyle = styles.imageLargePost;
+        colStyle = styles.col6;
+      } else if (mediaPosts.length === 2) {
+        colStyle = styles.col3;
+        if (index === 0) {
+          imageStyle = [styles.imageLargePost, styles.imageMarginRight];
```

Comment:
i'm not sure, can we rewrite this using css feature

Reply:
@bonanaaaaaa  I don't understand what you mean.
This is the old code. Should I go deep and refactor?

Reply:
ic, maybe later